### ISSUE TITLE
Kick user from stream with REST

### ISSUE THREAD

**Issue by golgetahir** [ant-media/Ant-Media-Server#2870]
**Is your feature request related to a problem? Please describe.**
Can't kick user from the streams and room. 

**Describe the solution you'd like**
A REST call to kick some user from the conference room and 1 to many viewers. 

**Describe alternatives you've considered**
PublishStreamId can be stopped in the conference but user may still view the streams?

**Comment by MarianoFacundoArch** [comment]
VERY necessary.

**Comment by broundtree** [comment]
agree... VERY necessary.

**Comment by mekya** [comment]
Thank you for feedbacks. We'll schedule it for next release

**Comment by fabioginzel** [comment]
+1

**Comment by ThomasCarltonInPerson** [comment]
This feature is not even necessary. It's mandatory. The question is rather : how come ant media server doesn't have it from it's first version ? At least for security reasons. Imagine a confindential meeting and malicious person joins. This hapenned only recently when a dutch journalist went into EU minister's zoom meeting.

**Comment by mekya** [comment]
Thank you @ThomasCarltonInPerson for your comments, just accept that it's our/my bad. 

There are some technical difficulties in resolving this issue and we did not have time to think about that. 

1. We need to assign an id for each viewer or let the developer assign it 
2. We need to kick the user in the cluster environment as well.  

Anyway, I'm moving this issue to the top of the backlog.

**Comment by sandipan-bangalore** [comment]
yes, this is very necessary feature. Please do it asap. Thank you.

**Comment by mekya** [comment]
Thank you for your comment. @sandipan-bangalore .

Yes it's something that we need to start working on it.

**Comment by mekya** [comment]
@burak-58 , could you please write simple instructions how to kickoff the users in standalone and cluster mode?
